### ISSUE TITLE
Unable to set editor font size

### ISSUE THREAD

**Issue by AlmeroSteyn** [xdrop/night-owl-jetbrains#17]
Thank you for this great theme.

Maybe it is just me but when I set the font size in *Settings | Editor | Font | Size*  it is totally overridden by the plugin. 

Does the plugin currently allow for this? If not it would be great if it could.

I am using scrolling with the mouse wheel to up the size now but have to do it every time I open a new file.

**Comment by xdrop** [comment]
Hey, I don't believe we do anything special regarding the font size.

Can you make sure the box below is *not* ticked?
<img width="449" alt="screen shot 2018-08-14 at 11 14 16" src="https://user-images.githubusercontent.com/12696953/44079968-4719fefe-9fb3-11e8-961c-b39a378c032c.png">

**Comment by AlmeroSteyn** [comment]
@xdrop, thank you so much for responding so quickly.

I have no idea why I did not see this setting. Maybe because I just uninstalled everything and reinstalled it but there it was. Set the font size there and it worked. I was changing font sizes just about everywhere else.

So issue on my side but you really lead me right to fixing it. Thank you so much. This theme is an absolute pleasure to look at and read.
